core/src/main/java/io/cloudevents/core/v03/CloudEventV03.java

```diff
@@ -36,6 +36,16 @@
  */
 public final class CloudEventV03 extends BaseCloudEvent {
 
+    private final static String ID = "id";
+    private final static String SOURCE = "source";
+    private final static String SPECVERSION = "specversion";
+    private final static String TYPE = "type";
+    private final static String TIME = "time";
+    private final static String SCHEMAURL = "schemaurl";
+    private final static String DATACONTENTTYPE = "datacontenttype";
+    private final static String DATACONTENTENCODING = "datacontentencoding";
+    private final static String SUBJECT = "subject";
```

Comment:
Can you make those one public?

Reply:
https://github.com/cloudevents/sdk-java/pull/282#issuecomment-729606022

Reply:
We can hardcode them or if you make those ones public we can use them.